ccd-definition/CaseField/CareSupervision/manageHearingsFields.json

```diff
@@ -84,6 +84,24 @@
     "HintText": "Use 24 hour format",
     "SecurityClassification": "Public"
   },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "hearingStartDateConfirmation",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "FieldType": "DateTime",
+    "Label": "Start date and time - new",
+    "HintText": "Use 24 hour format",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "hearingEndDateConfirmation",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "FieldType": "DateTime",
+    "Label": "End date and time - new",
+    "HintText": "Use 24 hour format",
+    "SecurityClassification": "Public"
+  },
```

Comment:
I will review properly later but all new fields should have "Searchable": "N" to avoid ES indexing.